transmart-server/grails-app/conf/logback.groovy

```diff
@@ -0,0 +1,39 @@
+import grails.util.BuildSettings
```

Comment:
This is the auto generated logback conf. Probably better to copy the one from transmartApp, as that has some extra stuff in it. (I presume grails will take the logback.groovy from the main application module)